array/DNAm/preprocessing/jobSubmission/1_runDNAmQC.sh

```diff
@@ -65,4 +66,13 @@ Rscript CETYGOdeconvolution.r ${DATADIR}
 
 ## print finish date and time
 echo Job finished on:
-date -u
\ No newline at end of file
+date -u
+
+mkdir -p ${DATADIR}/logFiles
+
+mv "${SLURM_SUBMIT_DIR}/QCDNAdata_${SLURM_JOB_ID}.log" \
+"${DATADIR}/logFiles/QCDNAdata_${SLURM_JOB_ID}.log"
+mv "${SLURM_SUBMIT_DIR}/QCDNAdata_${SLURM_JOB_ID}.err" \
+"${DATADIR}/logFiles/QCDNAdata_${SLURM_JOB_ID}.err"
```

Comment:
There's not really a good way around this though. Maybe worth putting into READMEs if this ever becomes relevant.

Reply:
Yes I did think this. That's partly why I include the print statements so the user knows where to look for the log files. I didn't have any obvious ideas how to avoid this?

Reply:
I mean it's not a problem at all for this script as the only way it will exit early is if the user cancels the job.

It usually becomes a problem when you have fail safe exit statements like:

```bash
cd some/directory/change || exit 1
```

Also, the print statements are kinda useless no? They will only ever be in the log files themselves. So if you are reading the line that tells you where the log file is, you've already found it.

Reply:
oh yeah!